src/applications/hca/tests/unit/utils/helpers/form-config.unit.spec.js

```diff
@@ -160,7 +166,12 @@ describe('hca form config helpers', () => {
     });
 
     it('should return `false` when user is short form eligible', () => {
-      const formData = getData({ disabilityRating: 80 });
```

Comment:
This test data was unrealistic, because you wont have a disability rating number as a guest users. Making the test data match what the user would have input.